modules/yarn.lock

```diff
@@ -7270,7 +7270,7 @@ discontinuous-range@1.0.0:
   resolved "https://registry.yarnpkg.com/discontinuous-range/-/discontinuous-range-1.0.0.tgz#e38331f0844bba49b9a9cb71c771585aab1bc65a"
   integrity sha1-44Mx8IRLukm5qctxx3FYWqsbxlo=
 
-dnd-core@^10.0.1:
+dnd-core@10.0.1, dnd-core@^10.0.1:
```

Comment:
makes me wonder where this comes from... 🤔

Reply:
I've seen this particular line flipping back and forth. I am suspicious that it is very sensitive to local state on the developer machine.

It [conflicted](https://github.com/brianchandotcom/liferay-portal/pull/91298) anyway with [my change](https://github.com/brianchandotcom/liferay-portal/pull/91296). For that reason, unless I think I'm the only one touching the `yarn.lock` file, I keep stuff like this (ie. stuff that doesn't change the actual resolution) out of the PR.

Reply:
> It conflicted anyway with my change. For that reason, unless I think I'm the only one touching the yarn.lock file, I keep stuff like this (ie. stuff that doesn't change the actual resolution) out of the PR.

😂 Yeah... less is always more, @diegonvs! 😉